source/Octopus.Tentacle.Tests.Integration/ClientScriptExecutionRetriesTimeout.cs

```diff
@@ -98,7 +103,66 @@ await syncOrAsyncHalibut.WhenSync(() => scriptServiceV2.EnsureTentacleIsConnecte
             scriptServiceV2CallCounts.StartScriptCallCountStarted.Should().Be(0, "Test should not have not proceeded past GetCapabilities");
 
             // Ensure we actually waited and retried until the timeout policy kicked in
-            duration.Elapsed.Should().BeGreaterOrEqualTo(TimeSpan.FromSeconds(14));
+            duration.Elapsed.Should().BeGreaterOrEqualTo(clientAndTentacle.RpcRetrySettings.RetryDuration - retryIfRemainingDurationAtLeastBuffer - retryBackoffBuffer);
+
+            RetryLogMessageAssertions.AssertRetryAttemptsLoggedAndRetryFailureLogged(inMemoryLog);
+        }
+
+        [Test]
+        [TentacleConfigurations]
+        public async Task WhenGetCapabilitiesFails_AndTakesLongerThanTheRetryDuration_TheCallIsNotRetried_AndTimesOut(TentacleConfigurationTestCase tentacleConfigurationTestCase)
```

Comment:
Additional testing to cover the scenario where the initial request takes longer than the retry duration